### ISSUE TITLE
Plugin does not honor the maven proxy settings

### ISSUE THREAD

**Issue by reinhapa** [eXist-db/public-xar-repo-plugin#7]
When using the plugin behind a proxy as configured in the maven `settings.xml` file:
```xml
<settings>
  <proxies>
    <proxy>
      <id>http-proxy</id>
      <active>true</active>
      <protocol>http</protocol>
      <host>127.0.0.1</host>
      <port>3128</port>
      <nonProxyHosts>localhost|127.0.0.*|10.*|192.168.*</nonProxyHosts>
    </proxy>
    <proxy>
      <id>https-proxy</id>
      <active>true</active>
      <protocol>https</protocol>
      <host>127.0.0.1</host>
      <port>3128</port>
      <nonProxyHosts>localhost|127.0.0.*|10.*|192.168.*</nonProxyHosts>
    </proxy>
  </proxies>
</settings>
```

The actual download of extensions will timeout:
```
[INFO] --- public-xar-repo-plugin:1.0.2:resolve (fetch-xars) @ exist-distribution ---
[INFO] Attempting to resolve package: {"abbrev": "dashboard"}
[INFO] 
Jun 24, 2019 4:32:46 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.net.SocketException) caught when processing request to {}->http://exist-db.org:80: Network is unreachable (connect failed)
Jun 24, 2019 4:32:46 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}->http://exist-db.org:80
Jun 24, 2019 4:34:57 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.net.SocketException) caught when processing request to {}->http://exist-db.org:80: Network is unreachable (connect failed)
Jun 24, 2019 4:34:57 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}->http://exist-db.org:80
Jun 24, 2019 4:37:08 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.net.SocketException) caught when processing request to {}->http://exist-db.org:80: Network is unreachable (connect failed)
Jun 24, 2019 4:37:08 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}->http://exist-db.org:80
```

**Comment by dizzzz** [comment]
Interesting.... did not see this.

**Comment by adamretter** [comment]
@reinhapa I just published version `1.1.0` which should honour the proxy settings in your Maven `settings.xml` file.
